doc/exclude_patterns.inc

```diff
@@ -5,12 +5,12 @@ modules/core/modules/otpinfo
 modules/core/modules/random
 modules/core/modules/trycatch
 modules/rtos/modules/FreeRTOS
-modules/inferencing
 modules/io/modules/xud
 modules/io/doc/substitutions.rst
 modules/sample_rate_conversion
-modules/voice/doc/substitutions.rst
+modules/lib_qspi_fast_read
 modules/xscope_fileio
+modules/sw_pll
```

Comment:
Why has the SW PLL documentation been excluded?

Reply:
We never really included it, it's not in the toctree, but doxygen still tries to build it and throws a lot warnings